<h1>Table of Contents<span class="tocSkip"></span></h1>
<div class="toc"><ul class="toc-item"><li><span><a href="#Example-1" data-toc-modified-id="Example-1-1"><span class="toc-item-num">1&nbsp;&nbsp;</span>Example 1</a></span><ul class="toc-item"><li><span><a href="#Interpretation-of-result" data-toc-modified-id="Interpretation-of-result-1.1"><span class="toc-item-num">1.1&nbsp;&nbsp;</span>Interpretation of result</a></span></li></ul></li><li><span><a href="#Example-2" data-toc-modified-id="Example-2-2"><span class="toc-item-num">2&nbsp;&nbsp;</span>Example 2</a></span><ul class="toc-item"><li><span><a href="#Interpretation-of-result" data-toc-modified-id="Interpretation-of-result-2.1"><span class="toc-item-num">2.1&nbsp;&nbsp;</span>Interpretation of result</a></span></li></ul></li></ul></div>

## Example 1

Given the following data calculate a 95% CI

$n = 15$

$s = 4$

$\bar x = 20$

The confidence interval is given by the follwing formula


<span style="font-size: 16px; color: blue;"> $\bar{X} - t_\frac{\alpha}{2}.\frac{s}{\sqrt{n}} < \mu < \bar{X} + t_\frac{\alpha}{2}.\frac{s}{\sqrt{n}}$</span>


In [4]:
# step 1 find t for alpha of 0.025 and 0.975

t = qt(c(0.025,0.975), df = 14)
t

# step 2 find s_x

s_x = 4/sqrt(15)
s_x

margin_of_error <- t*s_x
margin_of_error

# step 3 calculate ci
ci <- 20 + margin_of_error
ci

[1] -2.144787  2.144787

[1] 1.032796

[1] -2.215126  2.215126

[1] 17.78487 22.21513

### Interpretation of result

We are 95% confident the the interval (17.79 - 22.21) contains the population mean.

We're not saying that there is a 95% probability that the population mean is inside the interval.

## Example 2

Given the following data, calculate a 95% CI

$n = 30$

$s = 1958$

$\bar x = 3661.5$

The confidence interval is given by the follwing formula


<span style="font-size: 16px; color: blue;"> $\bar{X} - t_\frac{\alpha}{2}.\frac{s}{\sqrt{n}} < \mu < \bar{X} + t_\frac{\alpha}{2}.\frac{s}{\sqrt{n}}$</span>

In [6]:
t = qt(c(0.025,0.975), df = 29)
t

# step 2 find s_x

s_x = 1958/sqrt(30)
s_x

margin_of_error <- t*s_x
margin_of_error

# step 3 calculate ci
ci <- 3361.5 + margin_of_error
ci

[1] -2.04523  2.04523

[1] 357.4803

[1] -731.1292  731.1292

[1] 2630.371 4092.629

### Interpretation of result

We are 95% confident that salesperson spend between 2630 and 4092 seconds with a false customer each 8 hour day. Or we are 95 confident that this interval contains the population mean.